omnigibson/controllers/controller_base.py

```diff
@@ -125,13 +126,13 @@ def __init__(
         self._command_input_transform = None
 
         # Standardize command input / output limits to be (min_array, max_array)
-        command_input_limits = (-1.0, 1.0) if command_input_limits == "default" else command_input_limits
+        command_input_limits = (-1.0, 1.0) if MAGIC_DEFAULT == command_input_limits else command_input_limits
```

Comment:
Remove all the references to this MAGIC_DEFAULT. Instead, put this check in: `type(command_input_limits) == str and command_input_limits == "default"`. Do this for all of the examples below.